dso-l2/src/main/java/com/tc/services/TerracottaServiceProviderRegistryImpl.java

```diff
@@ -53,27 +53,12 @@ public void initialize(String serverName, TcConfiguration configuration) {
             registerNewServiceProvider(provider);
           }
         } catch (InstantiationException | IllegalAccessException ie) {
-//  really shouldn't be doing this.  ServiceProvider configurations should provide appropriate classes
-//  to handle the config.
-          tryServiceLoader(config);
+          logger.error("caught exception while initializing service " + serviceClazz, ie);
```

Comment:
This is a good idea:  I am worried that this is a serious problem which we are going to ignore and relegate to logs.

We should re-throw, probably.  I would want to see the server fail start-up.